config/webpack/alias-builder.js

```diff
@@ -46,8 +46,14 @@ module.exports = {
     // Allow SPAs to provide custom module aliases.
     const moduleAliases = skyPagesConfig.skyux.moduleAliases;
     if (moduleAliases) {
+      const command = (argv && argv.command) ? argv.command : 'serve';
```

Comment:
I believe there's a test-case here we're not currently covering.  I would be comfortable relying on `argv` always existing (as it does in real code) and just updated the test suite to make sure to pass it in.